### PR TITLE
Added introduction about myself (user Nallikaea)

### PR DESCRIPTION
**Pull request by nallikaea**
To finish learning "Git intro" seqtion I added introduction about myself and picture of my cat.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1513,4 +1513,10 @@ And meet my cat who saw some things in this world :D
 
 ![Cat](assets/images/frustrating-tisha.jpg)
 
+---
+## Hello, Kottans!
```
Comment (Temu4) [inline README.md:1517]: Hey Maryna!

Great job! 👍 

I see that you try to use different markdown content (headers, bold etc.). And if you want to apply them here you have to add a new line between content sections.

**Review by Temu4 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Temu4) [reply README.md:1517]: UPD: you can use different tools to preview you markdown in realtime (for example, for VSCode `shift+cmd+v` (`ctrl+v` for Windows).
Please, feel free to ping me you if you need more details.

Comment (nallikaea) [reply README.md:1518]: Thanx! I added a new line between content sections. Is it correct? )

Comment (nallikaea) [reply README.md:1518]: Hello, should I do something more? I added a new line. Is it correct? )

Comment (Temu4) [reply README.md:1518]: Great!
Looks good to me!

**Review by Temu4 — APPROVED** [review]

**Pull request merged**